packages/react-integration/demo-app-ts/src/components/demos/CardDemo/CardDemo.tsx

```diff
@@ -93,7 +93,7 @@ export class CardDemo extends React.Component {
 
     return (
       <React.Fragment>
-        <Card isHoverable>
+        <Card isSelectable selectableVariant="raised">
```

Comment:
Just a thought - could we combine these so you don't need separate props? Wondering if `isSelectable="raised"` could apply the new modifier.

Reply:
This is a good thought - I figured that adding the selectableVariant="raised" would mean that to opt into the new styling, they'd just need to add indicate they wanted to using the new prop. And eventually, when raised is the default, then it wont be necessary to specify that.